Backend-SignUp-Login#8

ManpreetBhuller opened the pull request:
User login and sign up with JWT authentication at server side.

ManpreetBhuller committed:
Backend-SignUp-Login#8

User login and sign up with jwt authentication a server side.

ncampbell89 committed:
First commit for this branch

ManpreetBhuller committed:
Merge branch 'clientModifications' of https://github.com/hatchways/team-donut into SignUp-Login-#8

bonnieli commented on server/routes/api/user.js:
```diff
@@ -0,0 +1,73 @@
+
+//api for user registeration/creation
+router.post('/register',
+[
```
what IDE are you using to write code? perhaps look into a linter. this will help code be more organized and easier to read :)

ManpreetBhuller committed:
updated-Signup&Login

Sign-Up/Login integrated with front end.
When user do login or sign-up then we get data in response to the browser.

ManpreetBhuller committed:
Formatted

Formatted nicely
